src/routes/balances/entities/token-info.entity.ts

```diff
@@ -1,10 +1,18 @@
+import { TokenInfo as DomainTokenInfo } from '../entities/token-info.entity';
+import { ApiProperty } from '@nestjs/swagger';
 import { TokenType } from './token-type.entity';
 
-export interface TokenInfo {
-  tokenType: TokenType;
+export class TokenInfo implements DomainTokenInfo {
+  @ApiProperty()
   address: string;
+  @ApiProperty()
   decimals: number;
-  symbol: string;
+  @ApiProperty()
+  logoUri: string;
+  @ApiProperty()
   name: string;
-  logoUri?: string;
```

Comment:
Are you aware about this, now `logoUri` is mandatory.

Reply:
Just to be clear, `logoUri` changed recently from an optional field to a non-optional field?

Reply:
Nope, I meant that was optional until this change. I want to be sure that you are aware about this.

Reply:
I think the problem is a bad import, this should be indeed implementing a domain entity interface 🙂